packages/components/TopMenu/TopMenuMyTeritories.tsx

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import { Image, StyleSheet, TouchableOpacity, View } from "react-native";
+
+import orgGnoPunksPNG from "../../../assets/default-images/orgGnopunks.png";
+import orgJungleDefendersPNG from "../../../assets/default-images/orgJungleDefenders.png";
+import { neutral00, neutral77 } from "../../utils/style/colors";
+import {
+  fontBold11,
+  fontMedium10,
+  fontSemibold14,
+} from "../../utils/style/fonts";
+import { layout, topMenuWidth } from "../../utils/style/layout";
+import { BrandText } from "../BrandText";
+import FlexCol from "../FlexCol";
+import FlexRow from "../FlexRow";
+import { TertiaryBox } from "../boxes/TertiaryBox";
+import { CustomPressable } from "../buttons/CustomPressable";
+import { SmallCarousel } from "../carousels/SmallCarousel";
+import { TopMenuSection } from "./TopMenuSection";
+
+//TODO: Make SmallCarousel with items auto sized instead of fix widths
+const orgCardWidth = 164;
+const orgCardTagWidth = 94;
+
+type FakeOrg = {
+  name: string;
+  image: string;
+  tag: string;
+};
+const fakeOrgs: FakeOrg[] = [
+  { name: "GNOPUNKS", tag: "@GNOPUNKS.teritori", image: orgGnoPunksPNG },
+  {
+    name: "Jungle Defenders",
+    tag: "@JungleDefenders.teritori",
+    image: orgJungleDefendersPNG,
+  },
+  { name: "Fake 3", tag: "@JungleDefenders.teritori", image: orgGnoPunksPNG },
+  { name: "Fake 4", tag: "@sqfsf.teritori", image: orgJungleDefendersPNG },
+  { name: "Fake 5", tag: "@FGDGDG_sgdgdgdi.teritori", image: orgGnoPunksPNG },
+];
+
+const OrgCard: React.FC<{ organization: FakeOrg }> = ({ organization }) => {
+  return (
+    <TouchableOpacity onPress={() => {}}>
+      <TertiaryBox
+        height={48}
```

Comment:
Suggestion: use ```padding_x``` if possible

Reply:
It used only for spacing (margin, padding, SpacerRow, Separator, etc..)